oadr/oadr/manager/VENManager.h

```diff
@@ -83,6 +83,7 @@ class VENManager : public ISendCreatedEvent, public ISendReport, public IVENMana
 	virtual ~VENManager();
 
 	static IVENManager *init(VENManagerConfig &config);
+	static IVENManager *init(VENManagerConfig &config, std::unique_ptr<IHttp> http);
```

Comment:
Unfortunately, I needed to expose that method for testing mocking purpose. VENManager is constructed in a way that makes it really hard to test. I believe that we should create a new class that will be responsible for initialization only and will have no logic, so the hard to test code will be reduced to minimum.